slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/DbSlashingProtection.java

```diff
@@ -146,9 +147,13 @@ public void exportDataWithFilter(final OutputStream output, final List<String> p
 
   @Override
   public IncrementalExporter createIncrementalExporter(final OutputStream out) {
+    if (!gvrValidator.genesisValidatorRootExists()) {
```

Comment:
Wondering if make sense to handle the non-incremental case as well?

Reply:
We can, instead of throwing RuntimeException, we can instead return an empty String (or populate metadata with empty gvr value).